code/modules/mob/living/carbon/human/human_dummy.dm

```diff
@@ -74,11 +74,42 @@ GLOBAL_LIST_EMPTY(dummy_mob_list)
 /mob/living/carbon/human/dummy/add_to_all_mob_huds()
 	return
 
-
 /mob/living/carbon/human/dummy/tutorial // Effectively an even more disabled dummy
 
 /mob/living/carbon/human/dummy/tutorial/Initialize(mapload)
 	. = ..()
 	status_flags = GODMODE
 	ADD_TRAIT(src, TRAIT_IMMOBILIZED, TRAIT_SOURCE_TUTORIAL)
 	anchored = TRUE
+
+// Used by the CMO and the SEA for teaching medical personnel
+/mob/living/carbon/human/dummy/professor_dummy/Initialize(mapload)
+	. = ..()
+
+	// Personalise it
+	change_real_name(src, "Professor DUMMY the Medical Mannequin")
+	age = rand(1,5)
+	gender = pick(MALE, FEMALE)
+	var/datum/preferences/A = new
+	A.randomize_appearance(src)
+	regenerate_icons()
+
+	// Ensure nurses can practice surgery on it
+	mob_flags |= EASY_SURGERY
```

Comment:
You are missing `huggable = FALSE` like `/datum/equipment_preset/other/professor_dummy/load_race` used to do.